### PR TITLE
Client Side Encryption : Adds fix for Encryption Package not picking up Cryptography DLL.

### PR DESCRIPTION
**Pull request by kr-santosh**
## Description

This fixes the issue of Encryption package not picking up Cryptography.dll when building the Nuget package.

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: Microsoft.Azure.Cosmos.Encryption/src/Microsoft.Azure.Cosmos.Encryption.csproj [hunk 1]
```diff
@@ -40,7 +40,11 @@
   <ItemGroup>
     <Folder Include="Custom\" />
   </ItemGroup>
-  
+	
+  <ItemGroup>
+    <None Include="..\src\Custom\MdeServices\MdeCryptographyLib\Microsoft.Data.Encryption.Cryptography.dll" Pack="true" PackagePath="lib\netstandard2.0" />
```
Comment (j82w) [inline Microsoft.Azure.Cosmos.Encryption/src/Microsoft.Azure.Cosmos.Encryption.csproj:45]: Please generate the package locally to verify it includes the nuget and that the nuget functions as expected.

Comment (kr-santosh) [reply Microsoft.Azure.Cosmos.Encryption/src/Microsoft.Azure.Cosmos.Encryption.csproj:45]: Hi Jake,

Verified it by creating a local package to confirm if the DLL was indeed getting copied to lib folder in the Nuget package and also by using it.

**Review by j82w — APPROVED** [review]

File: Microsoft.Azure.Cosmos.Encryption/changelog.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ Preview features are treated as a separate branch and will not be included in th
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### <a name="1.0.0-preview9"/> [1.0.0-preview9](https://www.nuget.org/packages/Microsoft.Azure.Cosmos.Encryption/1.0.0-preview9) - 2021-01-06
+
+#### Added
```
Comment (j82w) [inline Microsoft.Azure.Cosmos.Encryption/changelog.md:8]: nit: Fixes
```suggestion
#### Fixes 
```

**Review by abhijitpai — APPROVED** [review]

File: Microsoft.Azure.Cosmos.Encryption/changelog.md [hunk 3]
```diff
@@ -3,6 +3,11 @@ Preview features are treated as a separate branch and will not be included in th
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### <a name="1.0.0-preview9"/> [1.0.0-preview9](https://www.nuget.org/packages/Microsoft.Azure.Cosmos.Encryption/1.0.0-preview9) - 2021-01-06
+
+#### Added 
+- [#2105](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2105) Adds fix for Encryption Package not picking up Cryptography DLL.
```
Comment (j82w) [inline Microsoft.Azure.Cosmos.Encryption/changelog.md:9]: Fixes the nuget generation to include Cryptography DLL

Comment (kr-santosh) [reply Microsoft.Azure.Cosmos.Encryption/changelog.md]: Made the change

**Review by j82w — APPROVED** [review]

**Review by anujtoshniwal — APPROVED** [review]

**Review by abhijitpai — APPROVED** [review]

**Pull request merged**